src/Operations.jl

```diff
@@ -1228,7 +1229,10 @@ end
 
 testdir(source_path::String) = joinpath(source_path, "test")
 testfile(source_path::String) = joinpath(testdir(source_path), "runtests.jl")
-function test(ctx::Context, pkgs::Vector{PackageSpec}; coverage=false, test_fn=nothing)
+function test(ctx::Context, pkgs::Vector{PackageSpec}; coverage=false, test_fn=nothing, args=String[])
+    if isa(args, AbstractString)
```

Comment:
I know this is how `Base.runtests` does this, but its ugly and I think we should restrict to `Vector{String}`.

Reply:
Makes sense to me. But could we do `AbstractVector{<:AbstractString}` instead of `Vector{String}`?

Reply:
Sure.

Reply:
Done!

Reply:
(Will push shortly)